.gitignore

```diff
@@ -162,3 +162,9 @@ cython_debug/
 *.charm
 *.rock
 !examples/**/lib
+
+# Docs-related files
+docs/doc-cheat-sheet-myst.md
+docs/doc-cheat-sheet.rst
+docs/.sphinx/latex_elements_template.txt
```

Comment:
You don't want these to be committed?

Reply:
Thanks for catching, I added them here because I (incorrectly) thought that Vale would ignore the files listed here. I will revert these changes